recipe_scrapers/_utils.py

```diff
@@ -65,21 +73,35 @@
 
 def _extract_fractional(input_string: str) -> float:
     input_string = input_string.strip()
-    if "/" in input_string:
-        # for example "1 1/2" is matched
-        components = input_string.split(" ")
-        whole_part, fractional_part = components[0], components[-1]
+
+    # Handling mixed numbers with unicode fractions e.g., '1⅔'
+    for unicode_fraction, value in FRACTIONS.items():
+        if unicode_fraction in input_string:
+            parts = input_string.partition(unicode_fraction)
+            whole_number_part = parts[0]
+            fraction_part = value
+
+            whole_number = float(whole_number_part) if whole_number_part else 0
+            return whole_number + fraction_part
+
+    if input_string in FRACTIONS:
+        return FRACTIONS[input_string]
+
+    try:
+        return float(input_string)
+    except ValueError:
+        pass
+
+    if " " in input_string and "/" in input_string:
+        whole_part, fractional_part = input_string.split(" ", 1)
```

Comment:
Could you add some test coverage for this?

(it's to handle situations such as `2  1 / 3` or similar, I guess?)